libfuse/debug.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keybase/client/go/logger"
+)
+
+// MakeFuseDebugFn returns a function that logs its argument to the
+// given log, suitable to assign to fuse.Debug.
+func MakeFuseDebugFn(
+	log logger.Logger, superVerbose bool) func(msg interface{}) {
+	return func(msg interface{}) {
+		str := fmt.Sprintf("%s", msg)
+		// If superVerbose is not set, filter out Statfs and
```

Comment:
@tv42 Would it be possible to expose something in bazil that would let me filter more accurately? And also does bazil have a context it can pass in? (Although I guess that's more useful for replies than it is for requests...)